File: 2018-12-21/10871/10871.java
import java.io.*;

class Main{
	public static void main(String[] args) throws Exception{
		BufferedReader bfr = new BufferedReader(new InputStreamReader(System.in));
		String[] info = bfr.readLine().split(" ");
		StringBuilder strBuilder = new StringBuilder();

		int N = Integer.parseInt(info[0]);
		int X = Integer.parseInt(info[1]);
		String[] numbers = bfr.readLine().split(" ");
		int [] result = new int[N];
		int count = 0;
		for(int i =0; i <N; i++){
			if( X > Integer.parseInt(numbers[i])){
				strBuilder.append(numbers[i]).append(" ");
			}
		}
		System.out.println(strBuilder);
	}
}

File: 2018-12-21/10871/10871.c
#include <stdio.h>

main(){
	int X, N;
	int count = 0;
	int result[10000] = {0,};
	int inputValue = 0;
	scanf("%d %d", &X, &N);
	for(int i = 0; i < X;i++){
		scanf("%d", &inputValue);
		if(inputValue < N){
			result[count] = inputValue;
			count++;
		}
		
	}
	for(int i = 0; i< count;i++)
		printf("%d ",result[i]);
}